v0/ai-logic/cli.py

```
@@ -0,0 +1,476 @@
+import httpx
+import json
+import os
+from rich.console import Console
+from rich.table import Table
+from rich.panel import Panel
+from rich.prompt import Prompt, Confirm
+from rich.progress import Progress, SpinnerColumn, TextColumn
+from typing import Dict, Any, Optional, List
+
+from schemas import VideoRequest, VideoStyle
+from routing import ProviderRouter
```

Ruff (0.12.2)

`routing.ProviderRouter` imported but unused

Remove unused import: `routing.ProviderRouter`

(F401)